web-marketplace/src/pages/Projects/AllProjects/AllProjects.tsx

```diff
@@ -265,7 +274,33 @@ export const AllProjects: React.FC<React.PropsWithChildren<unknown>> = () => {
               purchaseDetailsTitles={purchaseDetailsTitles}
               buttons={buttons}
               useProjectCardButton={IS_REGEN}
-              hasBottomCard={IS_REGEN}
+              creditIcon={
+                <>
+                  <InfoTooltip
+                    arrow
+                    placement="top"
+                    title={_(VOLUNTARY_MARKET_TOOLTIP)}
+                  >
+                    <img src={tebuLogo} alt="tebu" className="ml-3" />
+                  </InfoTooltip>
```

Comment:
Shouldn't this be wrapped in a conditional, checking `isVoluntaryProject`? Currently it looks like a compliance project would render both the voluntary and compliance tooltip.